apps/daffio/src/app/docs/guide-docs/components/guide-nav/guide-nav.component.html

```diff
@@ -0,0 +1,13 @@
+<daff-accordion *ngIf="guideList">
```

Comment:
Doesn't need to be here. This section should just be scrollable.

Reply:
Do you mean that the `daff-accordion` doesn't need to be there or that the `*ngIf` doesn't need to be there or both?

Reply:
Nvm, I was confused with the structure. This can stay.